pyproject.toml

```diff
@@ -31,20 +34,23 @@ scikit-learn = "^1.2"
 omegaconf = "^2.3.0"
 cachetools = "^5.3.0"
 orjson = "^3.9.5"
+typer = {version = "^0.9", extras = ["rich"]}
+matplotlib = "^3.4.2"
+pynumaflow = "~0.4"
```

Comment:
Why do we want it here?
Can't we just have extras for the backtest-script (especially pynumaflow)?

Reply:
No, unfortunately it is not recommended to have extras in the cli tool, since we are exporting the library as an executable here.

Reply:
I see. Cool!